charts/incubator/privatebin/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+appVersion: "1.4.0"
+dependencies:
+  - name: common
+    repository: https://library-charts.truecharts.org
+    version: 10.1.1
+deprecated: false
+description: PrivateBin is a minimalist, open source online pastebin where the server has zero knowledge of pasted data.
+home: https://github.com/truecharts/apps/tree/master/charts/incubator/privatebin
+icon: https://truecharts.org/_static/img/appicons/privatebin.png
+keywords:
+  - privatebin
+  - pastebin
+kubeVersion: ">=1.16.0-0"
+maintainers:
+  - email: info@truecharts.org
+    name: TrueCharts
+    url: https://truecharts.org
+name: privatebin
+sources:
+  - https://privatebin.info/
+  - https://github.com/PrivateBin/PrivateBin
+  - https://hub.docker.com/r/privatebin/pdo  # Optimized image for database backends
+type: application
+version: 0.1.0
```

Comment:
```suggestion
version: 0.0.1
```

Reply:
I changed it, but it broke some of the checks, so I reverted it to `0.1.0`.